apps/web-twig-demo/templates/default.html.twig

```diff
@@ -4,6 +4,8 @@
 
 {% block content %}
 
+  {% include 'partials/tabs.html.twig' %}
```

Comment:
I think we should create a new layout for index pages where the header and the tabs are included. Thus we will be not repeating them on every page. What do you think?

Reply:
It should be part of `default` and components&helpers should use `plain`. Fixed.